src/normalize/entities.ts

```diff
@@ -54,8 +50,7 @@ const RecurrenceInfo = new Entity({
 });
 
 const CalendarItemAlarm = new Entity({
-	trigger: CalendarItemAlarmTrigger,
-	at: ['attendees', CalendarItemAlarmAttendees]
```

Comment:
Any reason to remove this? It seems like a valid property on an `<alarm>`.

Reply:
- [ ] let's put this back.  It's in the graphql schema for the output